src/routes/Root/Root.tsx

```diff
@@ -11,26 +10,12 @@ import './Root.css';
 
 const Root = () => {
   const { t, i18n } = useTranslation();
-
-  const links = [
-    { to: `/${i18n.language}`, text: t('home') },
-    { to: `/${i18n.language}/${t('slugs.profile.index')}`, text: t('profile') },
-    { to: `/${i18n.language}/${t('slugs.about-us')}`, text: t('about-us') },
-    { to: `/${i18n.language}/${t('slugs.about-us')}/${t('slugs.terms-of-service')}`, text: t('terms-of-service') },
-    {
-      to: `/${i18n.language}/${t('slugs.about-us')}/${t('slugs.accessibility-statement')}`,
-      text: t('accessibility-statement'),
-    },
-    { to: `/${i18n.language}/${t('slugs.about-us')}/${t('slugs.privacy-policy')}`, text: t('privacy-policy') },
-    { to: `/${i18n.language}/${t('slugs.about-us')}/${t('slugs.cookies')}`, text: t('cookies') },
-  ];
-
   return (
     <>
       <Helmet>
         <html lang={i18n.language} />
       </Helmet>
-      <header className="sticky top-0">
+      <header className="sticky top-0 z-10">
```

Comment:
Without the z-index class the ColorCards were hovering over the header while scrolling.